tools/ci-ts/test-helpers/common.ts

```diff
@@ -1,7 +1,13 @@
-import chalk from 'chalk'
 import { ethers } from 'ethers'
+import { ContractReceipt } from 'ethers/contract'
+import { assert } from 'chai'
+import ChainlinkClient from './chainlinkClient'
+import { EventDescription } from 'ethers/utils/interface'
+
 import 'source-map-support/register'
```

Comment:
We can actually get rid of this now, since Jest will register source maps for us